tests/integration/suite/actors/http/ttl.go

```diff
@@ -99,16 +102,22 @@ func (l *ttl) Run(t *testing.T, ctx context.Context) {
 
 	req, err := http.NewRequest(http.MethodPost, daprdURL+"/v1.0/actors/myactortype/myactorid/method/foo", nil)
 	require.NoError(t, err)
-	resp, err := client.Do(req)
-	require.NoError(t, err)
-	assert.NoError(t, resp.Body.Close())
-	assert.Equal(t, http.StatusOK, resp.StatusCode)
+
+	assert.EventuallyWithT(t, func(c *assert.CollectT) {
+		resp, err := client.Do(req)
+		require.NoError(c, err)
+		assert.NoError(c, resp.Body.Close())
+		assert.Equal(c, http.StatusOK, resp.StatusCode)
+	}, time.Second*10, time.Millisecond*100, "actor not ready")
```

Comment:
We should not need this.. if daprd has reported healthy (`l.daprd.WaitUntilRunning(t, ctx)` didn't fail the test) then this _must_ be successful on the first call. If this needs more than one try, then it is a bug in runtime.

Reply:
We need this because there can be  a delay before the actor runtime is ready.

The gRPC test was already doing that: https://github.com/dapr/dapr/blob/e4f89012f56022a3e0dec18725493fb0c549ed2b/tests/integration/suite/actors/grpc/ttl.go#L104-L111 (looks like you wrote that)

Reply:
We should probably delay reporting 2xx on `/v1.0/healthz` until actor runtime is also ready.

Reply:
But what if someone doesn't want to use actors? Using the actor runtime is optional.

Anyways, I think it should be explored, but in a separate PR.